.gitignore

```diff
@@ -27,3 +27,5 @@ public
 gatsby-types.d.ts
 
 .env
+
+package-lock.json
```

Comment:
package-lock.json should be git committed
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

Reply:
^^ but commimt the yarn.lock since we're using yarn